src/tabs/tab-header-group.component.ts

```diff
@@ -192,6 +198,10 @@ export class TabHeaderGroup implements AfterContentInit, OnDestroy {
 	ngAfterContentInit() {
 		this.selectedSubscriptionTracker.unsubscribe();
 
+		if (this.tabHeaderQuery) {
+			this.tabHeaderQuery.toArray().forEach(tabHeader => tabHeader.cacheActive = this.cacheActive);
```

Comment:
This should be done in a way that it updates with changes in `cacheActive` during the lifetime of the component.

Reply:
Fixed!